### PR TITLE
Install the Shipment + Service Points feature, Install the Service Points feature

### PR DESCRIPTION
**Pull request by AsonUnique**
#### Overview:
- Epic ticket: https://spryker.atlassian.net/browse/CC-30636
- IG ticket: https://spryker.atlassian.net/browse/CC-30648

### CODE REVIEW

File: _includes/pbc/all/install-features/202400.0/install-the-service-points-feature.md [hunk 1]
```diff
@@ -1074,6 +1109,33 @@ Make sure that, in Redis, data is displayed in the following format:
    "_timestamp": 1683216744.8334839
 }
 ```
+
+Make sure that the `service-type` trigger plugin works correctly:
+
+1. Fill the `spy_service_type` table with data.
+2. Run the `console publish:trigger-events -r service_type` command.
+3. Make sure that the `spy_service_type_storage` table has been filled with respective data.
+4. Make sure that, in your system, storage entries are displayed with `kv:service_type:{service_point_id}` mask.
```
Comment (abitskil) [inline _includes/pbc/all/install-features/202400.0/install-the-service-points-feature.md:1118]: * We have to change from "service_point_id" to the  "service_type_id".

File: _includes/pbc/all/install-features/202400.0/install-the-service-points-feature.md [hunk 2]
```diff
@@ -1074,6 +1109,33 @@ Make sure that, in Redis, data is displayed in the following format:
...
+
+Make sure that `service-type` synchronization plugin works correctly:
+
+1. Fill the `spy_service_type_storage` table with some data.
+2. Run the `console sync:data -r service_type` command.
+3. Make sure that, in your system, storage entries are displayed with the `kv:service_type:{service_point_id}` mask.
```
Comment (abitskil) [inline _includes/pbc/all/install-features/202400.0/install-the-service-points-feature.md:1124]: * We have to change from "service_point_id" to the "service_type_id".

**Review by abitskil — APPROVED** [review]

File: _includes/pbc/all/install-features/202400.0/install-the-service-points-feature.md [hunk 3]
```diff
@@ -1074,6 +1109,33 @@ Make sure that, in Redis, data is displayed in the following format:
    "_timestamp": 1683216744.8334839
 }
 ```
+
+Make sure that the `service-type` trigger plugin works correctly:
+
+1. Fill the `spy_service_type` table with data.
+2. Run the `console publish:trigger-events -r service_type` command.
+3. Make sure that the `spy_service_type_storage` table has been filled with respective data.
+4. Make sure that, in your system, storage entries are displayed with `kv:service_type:{service_type_id}` mask.
```
Comment (VadymSachenko) [inline _includes/pbc/all/install-features/202400.0/install-the-service-points-feature.md:1118]:
```suggestion
4. Make sure that, in your system, storage entries are displayed with the `kv:service_type:{service_type_id}` mask.
```

File: _includes/pbc/all/install-features/202400.0/install-the-service-points-feature.md [hunk 4]
```diff
@@ -1074,6 +1109,33 @@ Make sure that, in Redis, data is displayed in the following format:
...
+
+Make sure that `service-type` synchronization plugin works correctly:
```
Comment (VadymSachenko) [inline _includes/pbc/all/install-features/202400.0/install-the-service-points-feature.md:1120]:
```suggestion
Make sure that the `service-type` synchronization plugin works correctly:
```

File: _includes/pbc/all/install-features/202400.0/install-the-shipment-service-points-feature.md [hunk 5]
```diff
@@ -169,10 +195,56 @@ class ShipmentTypeStorageDependencyProvider extends SprykerShipmentTypeStorageDe
 
 {% info_block warningBox "Verification" %}
 
-Make sure that `shipment-type` storage data is expanded with the service type field:
+Make sure that `shipment-type` storage data is expanded with the service type UUID:
 
 1. Fill the `spy_shipment_type_service_point` tables with data.
 2. Run the `console publish:trigger-events -r shipment_type` command.
-3. Make sure that the `spy_shipment_type_storage.data` field includes `service_type` data.
+3. Make sure that the `spy_shipment_type_storage.data` field includes `service_type.uuid` data.
```
Comment (VadymSachenko) [inline _includes/pbc/all/install-features/202400.0/install-the-shipment-service-points-feature.md:202]:
```suggestion
3. Make sure that the `spy_shipment_type_storage.data` field includes the `service_type.uuid` data.
```

File: _includes/pbc/all/install-features/202400.0/install-the-shipment-service-points-feature.md [hunk 6]
```diff
@@ -169,10 +195,56 @@ class ShipmentTypeStorageDependencyProvider extends SprykerShipmentTypeStorageDe
...
+
+{% endinfo_block %}
+
+3. Enable relationship between `shipment-types` and `service-types` resources in the Storefront API by registering the following plugins:
+
+| PLUGIN                                               | SPECIFICATION                                                      | PREREQUISITES | NAMESPACE                                                                          |
+|------------------------------------------------------|--------------------------------------------------------------------|---------------|------------------------------------------------------------------------------------|
+| ServiceTypeByShipmentTypesResourceRelationshipPlugin | Adds `service-types` resource as relationship by `shipment-types`. |               | Spryker\Glue\ShipmentTypesServicePointsResourceRelationship\Plugin\GlueApplication |
```
Comment (VadymSachenko) [inline _includes/pbc/all/install-features/202400.0/install-the-shipment-service-points-feature.md:210]:
```suggestion
| ServiceTypeByShipmentTypesResourceRelationshipPlugin | Adds the `service-types` resource as a relationship by `shipment-types`. |               | Spryker\Glue\ShipmentTypesServicePointsResourceRelationship\Plugin\GlueApplication |
```

**Review by VadymSachenko — APPROVED** [review]

**Review by VadymSachenko — DISMISSED** [review]

**Review by VadymSachenko — APPROVED** [review]

**Pull request merged**